.sequelizerc

```diff
@@ -0,0 +1,10 @@
+require('@babel/register');
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('./src/config', 'config.js'),
+  'models-path': path.resolve('./src/models'),
+  'seeders-path': path.resolve('./src/seeders'),
+  'migrations-path': path.resolve('./src/migrations')
+}
```

Comment:
Move sequelize folders into one folder. You can name it `Sequelize`.For instance `src/sequelize/models`